strider/normalizer.py

```diff
@@ -13,6 +16,12 @@ async def get_types(self, curies):
                 params=dict(curie=curies)
             )
         types = []
+        if response.status_code >= 300:
+            LOGGER.warning(
+                f"Received {response.status_code} response from normalizer: "
+                + response.text
+            )
+            return types
         for c in curies:
```

Comment:
Maybe we should add a warning to the log if we don't get any results back from normalizer. If we do, we should also add a check to the test to make sure the log message exists.

Reply:
You mean also warn if we get an empty or null 200 response? Yes, that's a good idea.